installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -90,59 +90,9 @@ resource "null_resource" "chef_provision_jenkins_server" {
     ]
   }
 
-  provisioner "file" {
```

Comment:
@bleggett Got an error while deploying. 

> HTTP Request Returned 412 Precondition Failed: No such cookbook: git
> Error Resolving Cookbooks for Run List

Reply:
What scenario? With Scenario 3 I get the correct result.

Reply:
@bleggett I think this is already discussed her - https://github.com/tmobile/jazz-installer/pull/136
Please take a look.

Reply:
Right, I remember this. The way to fix this once and for all is to stop copying things to $HOME.

Because $HOME changes depending on the scenario and whether we run chef-client as sudo, and if we use home we can't avoid hardcoding a user-specific path into the chef cookbooks, and we definitely want to avoid that.

Going to test out a change to copy the cookbooks to `/tmp`, that should work in all cases and should be pretty agnostic for all POSIX systems.